src/xmipp/bindings/python/xmipp_base.py

```diff
@@ -320,7 +320,7 @@ def installEnvironCmd(name: str, requirementsFn: str, gpu=False):
         
         target = name + '.yml'
         commands = [] 
-        commands.append('conda env create --yes -f %s' % requirementsFn)
+        commands.append('conda env create -f %s || conda env update -f %s' % (requirementsFn, requirementsFn))
```

Comment:
--yes was removed, add

Reply:
Not required with this change.

Reply:
Ok, I understood you provide two options the first (newers, the second the olders)